inference-engine/src/transformations/src/ngraph_ops/broadcastload.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph_ops/broadcastload.hpp"
+
+#include <ngraph/runtime/host_tensor.hpp>
+#include <ngraph/runtime/reference/broadcast.hpp>
+
+using namespace std;
+using namespace ngraph;
+
+constexpr NodeTypeInfo op::BroadcastLoad::type_info;
+
+op::BroadcastLoad::BroadcastLoad(const Output<Node>& x, Shape shape)
+: Op({x}), output_shape(shape), broadcast_info(x.get_shape().size(), 0) {
+    constructor_validate_and_infer_types();
+}
+
+bool op::BroadcastLoad::visit_attributes(AttributeVisitor& visitor) {
+    return true;
+}
+
+std::shared_ptr<Node> op::BroadcastLoad::clone_with_new_inputs(const OutputVector& new_args) const {
```

Comment:
Please add CC macros from the "itt.hpp" header in order to support conditional compilation feature.